### DISCUSSION TITLE
GitHub Copilot has no response in JetBrains software

### DISCUSSION THREAD

**Discussion by caseylin2002** [community/community#31204] [Copilot Conversations] [answered]
When using GitHub Copilot in Jetbrains software, it may suddenly appear that GitHub Copilot does not prompt you. It is useless to log in again, disable it and re-enable it. You can only reopen the software in order to continue to use it.

**Comment by davecheney** [comment]
@allynlin I'm sorry you're having difficulty. I see you're using version 1.1.28.1744, please try upgrading to the latest version of the plugin.

**Reply by caseylin2002** [reply]
It is because there is a problem with the latest version that we fall back to version 1.1.28.1744. There is no problem with the latest version in version 1.1.28.1744.

**Reply by davecheney** [reply]
I strongly recommend not remaining on older versions of the extension. That is not a supported path.

**Reply by caseylin2002** [reply]
Upgraded to the latest version 1.1.30.1887

**Reply by caseylin2002** [reply]
But there is still a chance that it will suddenly suspend work.

**Reply by davecheney** [reply]
@allynlin https://github.com/orgs/community/discussions/30716 tracks proxy support for intellij. I think that's the best place to express your concerns.

**Comment by davecheney** [comment, marked as answer]
@allynlin please consult https://github.com/community/community/discussions/30716 and comment if necessary.
